ndm-exporter/collector/seachest.go

```diff
@@ -52,8 +52,23 @@ type SeachestCollector struct {
 // SeachestMetricData is the struct which holds the data from seachest library
 // corresponding to each blockdevice
 type SeachestMetricData struct {
-	SeachestIdentifier *seachest.Identifier
-	TempInfo           blockdevice.TemperatureInformation
+	SeachestIdentifier   *seachest.Identifier
+	TempInfo             blockdevice.TemperatureInformation
+	ModelNumber          string
+	SerialNumber         string
+	VendorID             string
+	FirmwareVersion      string
+	UUID                 string
```

Comment:
I think UUID reported by seachest is the WWN of the disk. In that case, the field should be renamed.